opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/TracingFilter.java

```diff
@@ -227,8 +231,13 @@ public void destroy() {
      * @param httpServletResponse response
      * @return whether request should be traced or not
      */
-    protected boolean isTraced(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse) {
-        return true;
+    protected boolean isNotTraced(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse) {
+        if (skipPattern != null) {
+            String url = httpServletRequest.getRequestURI().substring(httpServletRequest.getContextPath().length());
```

Comment:
agree documentation should be better. Do you agree about this? I think it is the most convenient solution to specify only URL path fragment.